user-org-service/app/controllers/usermanagement/UserController.java

```diff
@@ -15,27 +17,31 @@
  */
 public class UserController extends BaseController {
 
-  /**
-   * This method is used to create user
-   *
-   * @return Return a promise of created userResponse
-   */
-  public CompletionStage<Result> createUser() {
-    startTrace("createUser");
-    CompletionStage<Result> response = handleRequest(request(),null, UserActorOperations.CREATE_USER.getOperation());
-    endTrace("createUser");
-    return response;
-  }
+    /**
+     * This method is used to create user
+     *
+     * @return Return a promise of created userResponse
+     */
+    public CompletionStage<Result> createUser() {
+        startTrace("createUser");
+        CompletionStage<Result> response = handleRequest(request(), (request) -> {
+            JSONObject jsonObject = jsonifyRequestObject(request().body().asJson());
+            new UserModelValidator().validate(jsonObject, request().uri());
```

Comment:
what will happen when it throw exception.

Reply:
it will be caught by the BaseController: \handleRequest method.

Reply:
ok